relay-server/src/statsd.rs

```diff
@@ -875,6 +879,8 @@ impl CounterMetric for RelayCounters {
             RelayCounters::EventCorrupted => "event.corrupted",
             RelayCounters::EnvelopeAccepted => "event.accepted",
             RelayCounters::EnvelopeRejected => "event.rejected",
+            RelayCounters::EnvelopeItems => "event.items",
+            RelayCounters::EnvelopeItemByteSize => "event.items.size_bytes",
```

Comment:
nit: might be just me but when I read size I expect I can do `avg`, `max`, etc.

```suggestion
            RelayCounters::EnvelopeItemBytes => "event.item_bytes",
```